types/chrome-remote-interface.d.ts

```diff
@@ -0,0 +1,246 @@
+// https://github.com/cyrus-and/chrome-remote-interface/blob/master/lib/protocol.json
```

Comment:
Is there any difference in this file? I'd like to know the reason to provide the typedef ourselves. https://github.com/ChromeDevTools/devtools-protocol/blob/master/types/protocol.d.ts

Reply:
I was not aware of that definitions were there already 😭 Should we replace it with them?

Reply:
I think this typedef file has no problem👍However when the devtools-protocol has changed, please note to be possible we give up to maintain it and use their definition.

Reply:
Nice! Perhaps ought to be replaced at the some point in the future though.